ERCS/erc-x.md

```diff
@@ -55,6 +72,37 @@ For [ERC-20](./eip-20.md) non-compliant Vaults, the `share` method SHOULD NOT re
       type: address
 ```
 
+### Multi-Vaults
+Multi-vaults share a single `share` token with multiple entry points denominated in different `asset` tokens.
+
+Multi-vaults MUST implement the `share` method on each entry point. The entry points SHOULD NOT be ERC-20.
```

Comment:
This is a bit confusing as per above (and how it should be imo), all implementations adhering to this ERC should implement the `share` method. Just for Multi-Vaults, `share!=address(this)`